### ISSUE TITLE
Validate and check fail with Django 1.6 

### ISSUE THREAD

**Issue by rcooke** [bitmazk/django-calendarium#23]
I just created a virtual env, checked out this project and its dependencies and I get this check error:

```
(py27dj16)17:07 ~/source/church $ ./manage.py validate
CommandError: One or more models did not validate:

schedule.eventrelation: Accessor for field 'content_type' clashes with related field 'ContentType.eventrelation_set'. Add a related_name argument to the definit
ion for 'content_type'.

calendarium.eventrelation: Accessor for field 'content_type' clashes with related field 'ContentType.eventrelation_set'. Add a related_name argument to the defi
nition for 'content_type'.
```

**Comment by Tyrdall** [comment]
@rcooke Seems you have django-schedule installed. Please deinstall and try again.

**Comment by rcooke** [comment]
Thanks!

I looked at the history of the models file, and upon realizing it had not been changed for some time, got suspicious I had created the error myself.

I was just making a fresh django project to test it by its self when you posted.  

Thanks again!

I have django-schedule installed because I am trying to decide which to use for a free project for a local church.  I am doing a spike test and thought I could try both side by side.

**Comment by Tyrdall** [comment]
Great. Yes, we used the schedule code to kickstart this application. =) FYI: schedule hasn't been updated for five years.

**Comment by rcooke** [comment]
Yes, your not the only person to feel that way.  Another team resurrected it as django-scheduler - note the "r" on the end.

I decided your project and their project were my "best bets" after studying this table:
https://www.djangopackages.com/grids/g/calendar/

You might want to update the feed to this page, I think you have advanced further along than what is shown here.

I got the sample views up.  But I'm not getting the answers I wanted.  I want to know how easily this package handles recurring events with exceptions.  For example: "2nd Tuesday of the month, except xx date was canceled, and yy date has been moved to a new date or room".
